src/app/booking/components/ReservationFormFields.tsx

```diff
@@ -145,31 +200,108 @@ const ReservationFormFields = ({ initialData, items, groups, groupChangeCallback
                         </FormItem>
                     )}
                 />
+                {selectedItem?.allows_alcohol && (
+                    <>
+                        <FormField
+                            control={form.control}
+                            name="sober_watch"
+                            render={({ field: { value, ...rest } }) => (
+                                <FormItem className="flex items-center space-x-2">
+                                    <FormControl>
+                                        <Checkbox
+                                            {...rest}
+                                            checked={value}
+                                            onCheckedChange={(e) =>
+                                                form.setValue(
+                                                    'sober_watch',
+                                                    Boolean(e.valueOf()),
+                                                    {
+                                                        shouldDirty: true,
+                                                        shouldTouch: true,
+                                                    },
+                                                )
+                                            }
+                                        />
+                                    </FormControl>
+                                    <FormLabel className="!my-0">
+                                        Vi skal servere alkohol på arrangementet
+                                        og har fyllt ut{' '}
```

Comment:
```suggestion
                                        og har fylt ut{' '}
```

![image](https://github.com/TIHLDE/kontresv2/assets/33499052/bc8456dd-24ac-4601-b445-f3c51515f4aa)
hvis du fjerner en `l`, så får jeg sovet litt bedre 😇